system/lib/wasmfs/js_api.cpp

```diff
@@ -126,19 +126,11 @@ int _wasmfs_mkdir(char* path, int mode) {
 int _wasmfs_rmdir(char* path){ return __syscall_unlinkat(AT_FDCWD, (intptr_t)path, AT_REMOVEDIR); }
 
 int _wasmfs_open(char* path, int flags, mode_t mode) {
-  int err = __syscall_openat(AT_FDCWD, (intptr_t)path, flags, mode);
-  if (err == -1) {
-    return -errno;
-  }
-  return err;
+  return __syscall_openat(AT_FDCWD, (intptr_t)path, flags, mode);
```

Comment:
Are these bugfixes?

Reply:
We realized that this check isn't actually needed, since the syscalls do not actually set errno. The tests were passing before because `err` was never equal to -1, so the `js_api.cpp` function directly returned the error without using errno.

Reply:
Can you make a separate PR for this cleanup stuff?